README.md

```diff
@@ -75,36 +70,75 @@ Default is github-pr-check. github-pr-review can use Markdown and add a link to
 
 **Optional**. Tool name to use for reviewdog reporter. Default = `remark-lint`.
 
-## Development
-
-### Release
-
-#### [haya14busa/action-bumpr](https://github.com/haya14busa/action-bumpr)
+## Docker input args
 
-You can bump version on merging Pull Requests with specific labels (bump:major,bump:minor,bump:patch).
-Pushing tag manually by yourself also work.
+Besides the aforementioned input arguments you can also supply additional input arguments for the remark-lint linter using the args keyword [run.args](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#runsargs).
 
-#### [haya14busa/action-update-semver](https://github.com/haya14busa/action-update-semver)
-
-This action updates major/minor release tags on a tag push. e.g. Update v1 and v1.2 tag when released v1.2.3.
-ref: <https://help.github.com/en/articles/about-actions#versioning-your-action>
-
-### Lint - reviewdog integration
-
-This reviewdog action template itself is integrated with reviewdog to run lints
-which is useful for Docker container based actions.
+```yaml
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args: ". --verbose"
+```
 
-![reviewdog integration](https://user-images.githubusercontent.com/3797062/72735107-7fbb9600-3bde-11ea-8087-12af76e7ee6f.png)
+## Advance use cases
 
-Supported linters:
+This action can be combined with [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) or [stefanzweifel/git-auto-commit-action](https://github.com/stefanzweifel/git-auto-commit-action) to also apply the annotated changes to the repository.
```

Comment:
Can you remove this feature or use reviewdog -f=diff instead to support code suggestion instead of committing whole changes w/o reviewdog?

If it just runs the formatter and don't need reviewdog, you should create a different action which doesn't require reviewdog and  users should use the action instead.

You can refer to https://github.com/reviewdog/action-suggester

Reply:
@haya14busa I now changed the action such that the formatting action is seperate from the annotation action. As a result, the formatting is only done when the user explicitly asks for the files to be formatted. Consequently, review dog only receives the unformatted output. What do you think about this implementation?

Reply:
@haya14busa I agree that commiting back the changes to the pull request or repository is bad practice. I use this feature to format + annotate the code while also createing a pull-request with the formatted code. We can also remove the [stefanzweifel/git-auto-commit-action](https://github.com/stefanzweifel/git-auto-commit-action) action from the README and keep the [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) action.

Reply:
I mean, it's fine to create an action that format the file and use commit / pull-request actions, but in this case, you don't need to use reviewdog, so it should be better to create a separate repo.
Users can just install or run simple formatted action and then use commit / pull-request actions instead of using this reviewdog action.

My second point is that reviewdog supports code-suggestion based on diff (see action-suggester for example as written above), so if you want to support this feature, I'm very much welcome to have such feature in reviewdog actions.

Reply:
@haya14busa Ah I see! I will revert the formatting related changes (both in this repository and in the [action-black](https://github.com/reviewdog/action-black) repository) and only add the new `remark_flags` argument. I will try out if I can use the [https://github.com/reviewdog/action-suggester](reviewdog/action-suggester ) with my actions and add a small note to this repository pointing to that action if people want to also autoformat their code.